catroid/src/org/catrobat/catroid/ProjectManager.java

```diff
@@ -504,6 +517,374 @@ private void correctAllNestedReferences(Script script) {
 		}
 	}
 
+	public void mergeProjectInCurrentProject(String projectNameToMergeFrom, FragmentActivity activity) {
+
```

Comment:
Remove all those unnecessary newlines after a method declaration to be consistent. Especially with *your own* code :wink: